org.expresslrs.ExpressLRSConfigurator.appdata.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.expresslrs.ExpressLRSConfigurator</id>
+  <name>ExpressLRS Configurator</name>
+  <project_license>GPL-3.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>ExpressLRS</developer_name>
+  <summary>Cross platform configuration tool for the ExpressLRS firmware</summary>
+  <url type="homepage">https://www.expresslrs.org/</url>
+  <url type="donation">https://opencollective.com/expresslrs</url>
+  <url type="bugtracker">https://github.com/ExpressLRS/ExpressLRS-Configurator/issues</url>
+  <url type="faq">https://www.expresslrs.org/3.0/faq/</url>
+  <url type="contribute">https://github.com/ExpressLRS/ExpressLRS-Configurator#starting-development</url>
+  <url type="vcs-browser">https://github.com/ExpressLRS/ExpressLRS-Configurator</url>
+  <description>
+    <p>
+      ExpressLRS-Configurator is a cross platform configuration tool for the ExpressLRS firmware.
+      Allowing you to configure your ExpressLRS radio module and bind it to your transmitter. Or just keep them up to date.
+    </p>
+  </description>
+  <keywords>
+    <keyword>radio</keyword>
+    <keyword>radio control</keyword>
+    <keyword>fpv</keyword>
+    <keyword>drone</keyword>
+  </keywords>
+  <launchable type="desktop-id">org.expresslrs.ExpressLRSConfigurator.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/razzeee/org.expresslrs.ExpressLRSConfigurator/master/screenshots/expresslrs-configurator-1.png</image>
```

Comment:
you shouldn't link to `master` but to a specific revision. (until you link to the flathub repo)

Reply:
Changed to link to a tag